data/accounts/index.js

```diff
@@ -343,4 +343,28 @@ export default [
     image: '/logos/accounts/stakeDAO.svg',
     environment: 'mainnet',
   },
+  {
+    address: '0xB8D85A5e0c5e3354d35d576818B57AEd0300dE16',
+    name: 'Prime Protocol Moonbeam',
+    image: '/logos/accounts/Prime_Logo_Blue_SVG_NEWBLUE.svg',
```

Comment:
```suggestion
    image: '/logos/accounts/prime.svg',
```
Perhaps just `prime.svg` for simplicity?